fudge-ratio/src/main/webapp/calculator/index.html

```diff
@@ -46,7 +46,7 @@ <h1>Fudge Ratio Calculator</h1>
             <p> </p>
         </div>
 
-        <a href="ActivitiesManager.html"> Go to activities Manager </a>
+        <a href="/activitiesManager.html"> Go to activities Manager </a>
```

Comment:
in general, we name things lowercase in a url, so is gmail.com not Gmail.com or facebook.com instead of FaceBook.com